boards/arm/mps3_an547/doc/index.rst

```diff
@@ -0,0 +1,191 @@
+.. _mps3_an547_board:
+
+ARM MPS3 AN547
+###############
+
+Overview
+********
+
+The mps3_an547 board configuration is used by Zephyr applications that run
+on the MPS3 AN547 board. It provides support for the MPS3 AN547 ARM Cortex-M55
+CPU and the following devices:
+
+- Nested Vectored Interrupt Controller (NVIC)
+- System Tick System Clock (SYSTICK)
+- Cortex-M System Design Kit GPIO
+- Cortex-M System Design Kit UART
+- Ethos-U55 NPU
+
+.. image:: img/mps3_an547.png
+     :width: 666px
+     :align: center
+     :height: 546px
+     :alt: ARM MPS3 AN547
+
+More information about the board can be found at the `MPS3 FPGA Website`_.
+
+Hardware
+********
+
+ARM MPS3 AN547 provides the following hardware components:
+
+- ARM Cortex-M55
+- Soft Macro Model (SMM) implementation of SSE-300 subsystem
+- Memory
+
+  - 8MB BRAM
+  - 4GB DDR4 SODIMM (by default, upgradeable to 8GB)
+  - 16GB eMMC
+  - 8MB QSPI Flash
+
+- Debug
+
+  - P‐JTAG, F-JTAG, SWD, 4-bit trace, 16-bit trace
+  - Four serial ports over USB
+
+- Interface
+
+  - AHB GPIO
+  - UART
+  - SPI
+  - I2C
+  - I2S
+  - Color LCD serial interface
+  - Ethernet
+  - VGA
+
+- On-board Peripherals
+
+  - Color LCD
+  - 10 LEDs
+  - 8 Switches
+  - 2 user push buttons
+
+Supported Features
+===================
+
+The mps3_an547 board configuration supports the following hardware features:
+
++-----------+------------+-------------------------------------+
+| Interface | Controller | Driver/Component                    |
++===========+============+=====================================+
+| NVIC      | on-chip    | nested vector interrupt controller  |
++-----------+------------+-------------------------------------+
+| SYSTICK   | on-chip    | systick                             |
++-----------+------------+-------------------------------------+
+| UART      | on-chip    | serial port-polling;                |
+|           |            | serial port-interrupt               |
++-----------+------------+-------------------------------------+
+| GPIO      | on-chip    | gpio                                |
++-----------+------------+-------------------------------------+
+
+Other hardware features are not currently supported by the port.
+See the `MPS3 FPGA Website`_ for a complete list of MPS3 AN547 board hardware
+features.
+
+The default configuration can be found in the defconfig file:
+``boards/arm/mps3_an547/mps3_an547_defconfig``.
+
+For mode details refer to `MPS3 AN547 Technical Reference Manual (TRM)`_.
+
+Serial Port
+===========
+
+The MPS3 AN547 has six UARTs. The Zephyr console output by default, uses
+UART0, which is exposed over the Debug USB interface (J8).
```

Comment:
Is it worth mentionning the need to update the VCP drivers from FTDI for the quad-device chip to work properly on most OSes?

Reply:
I'd prefer not to get into such details as over time that will probably not be relevant any more, and is so OS dependent (for example MacOS 11.2 didn't need it, my Fedora Linux system was also ok).